src/main/MQ2ImGuiTools.h

```diff
@@ -30,6 +30,8 @@ void ImGuiTools_DrawWindowsMenu();
 void InitializeImGuiConsole();
 void ShutdownImGuiConsole();
 void UpdateImGuiConsole();
+bool MouseOverUpdate();
+void AlphaSetting(bool);
```

Comment:
Only public functions should go into this header. These are functions that are internal to the console implementation.